app/jobs/just_characterize_job.rb

```diff
@@ -0,0 +1,38 @@
+# Copy of https://github.com/samvera/curation_concerns/blob/v1.7.7/app/jobs/characterize_job.rb
+#   with call to CreateDerivativesJob removed
+#   and file cleanup added
+class JustCharacterizeJob < ActiveJob::Base
+  include CurationConcerns::Lockable
+
+  queue_as CurationConcerns.config.ingest_queue_name
+
+  # @param [FileSet] file_set
+  # @param [String] file_id identifier for a Hydra::PCDM::File
+  # @param [String, NilClass] filepath the cached file within the CurationConcerns.config.working_path
+  def perform(file_set, file_id, filepath = nil)
+    filename = CurationConcerns::WorkingDirectory.find_or_retrieve(file_id, file_set.id, filepath)
+    raise LoadError, "#{file_set.class.characterization_proxy} was not found" unless file_set.characterization_proxy?
+
+    # Prevent other jobs from trying to modify the FileSet at the same time
+    acquire_lock_for(file_set.id) do
+      Hydra::Works::CharacterizationService.run(file_set.characterization_proxy, filename)
+      Rails.logger.debug "Ran characterization on #{file_set.characterization_proxy.id} (#{file_set.characterization_proxy.mime_type})"
+      file_set.characterization_proxy.save!
+      file_set.update_index
+      file_set.parent.in_collections.each(&:update_index) if file_set.parent
+    end
+
+    File.delete(filename)
+    dir = File.split(filename).first
```

Comment:
Yep, totally good / fair question. This job will always pull a file out of fedora into a 'working' space, characterize it, and then delete the temporary file. Even in dev there is not a cache of fedora originals sitting around for this purpose. And now that I've deleted the parameter that allowed for a local file, we couldn't pass one in even if we had one.

The only possible conflict would be if some other job had pulled the file into the working space and was doing something to it when this one cleaned it up. That's a risk that I (uncharcteristicallly!) am willing to take, since we will run this very infrequently if ever again. The only place this job is called is from the 'recharacterize' rake task.

Here, I'll add another comment on the job and the task.